metaflow/plugins/kubernetes/kubernetes.py

```diff
@@ -281,6 +282,7 @@ def create_jobset(
                 "METAFLOW_INIT_SCRIPT", KUBERNETES_SANDBOX_INIT_SCRIPT
             )
             .environment_variable("METAFLOW_OTEL_ENDPOINT", OTEL_ENDPOINT)
+            .environment_variable("METAFLOW_KUBERNETES_NAMESPACE", KUBERNETES_NAMESPACE)
```

Comment:
Instead of importing that, we can rely on setting 
```suggestion
            .environment_variable("METAFLOW_KUBERNETES_NAMESPACE", namespace)
```
since namespace is passed down to the function.

Reply:
Actually wait! Rather we will need selectors for namespace. Thats a much cleaner and k8s native approach of doing it.